transformations/src/main/java/jp/wasabeef/glide/transformations/BlurTransformation.java

```diff
@@ -102,6 +103,8 @@ public BlurTransformation(Context context, int radius, int sampling) {
     blur.forEach(output);
     output.copyTo(bitmap);
 
+    source.recycle();
+    resource.recycle();
```

Comment:
@wasabeef @start141 

I can fix the memory leak using `applicationContext`.
I fixed this PR.
Please check this.